EnglishFilter/sections/css_extended.txt

```diff
@@ -27,6 +27,10 @@ wormate.io#?#.line-center > .column-right:has(> div[id] > .label:contains(/^Adve
 wormate.io#?##game-cont > #results-view > div[id][style]:has(> .label:contains(/^Advertisement:$/))
 ! https://github.com/AdguardTeam/AdguardFilters/issues/59371
 pngegg.com#$?#.list_ads { remove: true; }
+! https://github.com/AdguardTeam/AdguardFilters/issues/60472
+duplichecker.com#?#div > div:has(> span > a[href="https://www.duplichecker.com/linksmgm1"])
+duplichecker.com#?#div > div:has(> div > div#ldo6uv2f > div > span > a[href="https://www.duplichecker.com/linksmgm1"])
+duplichecker.com#?#body > div.search_bar_area:has(> div > div > div > span > a[href="https://www.duplichecker.com/linksmgm1"])
```

Comment:
You should re-write it to something like these for leftovers:
```
duplichecker.com##div[class*="aadd_boox"]
duplichecker.com#?#body > div[class]:has(> div.container > div.clearfix + div[class] > div[class*="aadd_boox"])
duplichecker.com#?#.side-bar > div:not([class]):has(> div span.text-center > a[target="_blank"][rel="nofollow"])
```
`div > div` we don't use it in our filters, look at examples on `css_extended` rules. It should be more specific.
2nd rule - 2nd part. This `div#ldo6uv2f` seems to have a random id. You should avoid it in future.
3rd rule - 2nd part. Too much `div`, some of them you can skip.

Reply:
ok, omw
#ldo6uv2f seemed pretty static for me, i could change it to something like `div[id]`